test/transform.globalize.js

```diff
@@ -5,6 +5,8 @@ var mocha = require('mocha');
 var transform = require('../src/transform/globalize');
 
 mocha.describe('transform/globalize', function () {
+  var generatedModuleName = '__1dd241c4cd3fd1dd89c570cee98b79dd';
```

Comment:
It's a hash of `test.js` which is what is used as the file path in the tests. I believe in master that part is abstracted out.